src/csharp/Microsoft.AzureKinect.UnitTests/StructureTests.cs

```diff
@@ -8,9 +9,9 @@ namespace Microsoft.AzureKinect.UnitTests
     public class StructureTests
     {
         [Test]
-        public void Float2()
+        public void Vector2()
```

Comment:
This can probably all be removed. I don't think we need to test system libraries in our code.

Reply:
Will remove it then. Initially I've decided to keep them in place to check compatibility.